prjxray/tile.py

```diff
@@ -1,38 +1,251 @@
+""" Database files available for a tile type. """
 from collections import namedtuple
 import json
 from prjxray import lib
-""" Database files available for a tile """
+from prjxray.timing import fast_slow_tuple_to_corners, RcElement
+
 TileDbs = namedtuple(
     'TileDbs', 'segbits block_ram_segbits ppips mask tile_type')
 
-Pip = namedtuple(
-    'Pip', 'name net_to net_from can_invert is_directional is_pseudo')
-""" Site - Represents an instance of a site within a tile.
 
-name - Name of site within tile, instance specific.
-prefix - Prefix of site naming in Xilinx parlance.
-type - What type of slice this instance presents.
-pins - Instaces of site pins within this site and tile.  This is an tuple of
-       SitePin tuples, and is specific to this instance of the site within
-       the tile.
+class OutpinTiming(namedtuple('OutpinTiming', 'delays drive_resistance')):
```

Comment:
Should these be `OutPinTiming` / `InPinTiming` ?

Reply:
Fixed